FORKING.md

```diff
@@ -0,0 +1,85 @@
+
+## Den Spaß weggabeln :fork_and_knife:
+Damit du die Homepage bearbeiten kannst, musst du dir zunächst auf deinem GitHub-Account eine Kopie ("Fork") der Fachschaftsseite erstellen.
+Ruf dazu zunächst das [**Repository**](https://github.com/fsi-tue/cyberauftritt) auf. Klicke oben rechts auf "Fork". Du besitzt nun eine Kopie der Fachschaftswebseite. Klicke (auf der Seite deiner Kopie, die sich selbst öffnen sollte) auf den grünen Button "Clone or Download".
+
+
+### Mit GitHub Desktop
+> :information_source: Noch nicht ausprobiert. Freiwillige vor!
+
+### Mit SSH
+> Tipp: Wenn du nicht weißt, was SSH-Keys sind, gehe zum Unterpunkt "mit HTTPS". Begib dich direkt dorthin, gehe nicht über Los, ziehe keine 4000 DM ein.
+
+Kopiere die Zeile, die im Textfeld bei "Clone with SSH" steht. Öffne ein Terminal, gehe in das Verzeichnis in dem ein Unterordner für die Fachschaftsseite erstellt werden soll. Tippe dann im Terminal: `git clone <Strg-Shift-V>`
+Es sollte das hier entstehen:
+```bash
+git clone git@github.com:deinusername/cyberauftritt.git
+```
+Führe den Befehl aus. Der Ordner "cyberauftritt" entsteht.
+### Mit HTTPS
+Klicke auf "Use HTTPS". Kopiere die Zeile, die im Textfeld bei "Clone with HTTPS" steht. Öffne ein Terminal, gehe in das Verzeichnis in dem ein Unterordner für die Fachschaftsseite erstellt werden soll. Tippe dann im Terminal: `git clone <Strg-Shift-V>`
+Es sollte das hier entstehen:
+```bash
+git clone git@github.com:deinusername/cyberauftritt.git
+```
+Führe den Befehl aus. Du wirst nach GitHub-Benutzernamen und dem dazugehörigen Passwort gefragt.
+Der Ordner "cyberauftritt" entsteht.
+
+## Original-Repository als Referenz hinzufügen
+
+Je nach dem wie viel auf der Seite gerade geändert wird, musst du dafür sorgen, dass deine Kopie mit dem Original auf einem Stand bleibt. Hierzu musst du einen sog. "Upstream" hinzufügen.
+- `cd /[…]/cyberauftritt`
+### Mit SSH
+> Tipp: Wenn du nicht weißt, was SSH-Keys sind, gehe zum Unterpunkt "mit HTTPS". Begib dich direkt dorthin, gehe nicht über Los, ziehe keine 4000 DM ein.
+>
+```bash
+git remote add upstream git@github.com:fsi-tue/cyberauftritt.git
+```
+### Mit HTTPS
+```bash
+git remote add upstream https://github.com/fsi-tue/cyberauftritt
+```
+Benutzernamen und Passwort eingeben.
+
+## Änderungen holen und loslegen
+> :warning: Obacht! Prüfe mit `git status`, ob du Änderungen hast, die noch nicht committed wurden. Diese Schritte gehen davon aus, dass du einen "sauberen" Fork hast, also nichts das geändert, aber noch nicht gepusht wurde.
+
+- Änderungen vom Original holen: `git fetch upstream`
+- Eigene Kopie auf gleichen Stand bringen: `git merge upstream/master`
+
+Nun ist es erst mal ein ganz normaler Commit:
+
+- Zeugs machen, das du machen willst
+- `git commit -m '<explain stuff you did>'`
```

Comment:
Theoretisch noch ein `git add` davor (am besten mit `-p` oder `-i`).

Premium wäre ja `git commit --interactive -m '<explain stuff you did>'`, aber das dürfte ziemlich overkill für nen Anfänger sein :o

Da wahrscheinlich selbst `-p` zu kompliziert ist, sollten wir evtl. doch leider `-a` empfehlen (wobei selbst das noch keine neuen (untracked) Dateien aufnimmt - dafür bräuchten wir sonst ein extra `git-add` vor `git-commit`).
KP... Was mutet man Anfängern hier am besten zu?

Reply:
Optional: Evtl. https://chris.beams.io/posts/git-commit/ verlinken, dass Anfänger das gleich richtig lernen? Oder zumindest ein kurzes Beispiel für einen sinnvollen "Betreff"?